app/assets/javascripts/modules/dish/views/form.coffee

```diff
@@ -1,5 +1,9 @@
 @Lunchiatto.module 'Dish', (Dish, App, Backbone, Marionette, $, _) ->
   Dish.Form = Marionette.ItemView.extend
+    ERROR_MESSAGE: (maxDebt) ->
```

Comment:
I'm not sure if this is a valid JS syntax (I mean the underscore case)

Reply:
It turns out that the screaming case is a valid syntax for constants in JS (not valid here, since `ERROR_MESSAGE` is a function, so you're absolutely right!)